src/lib.rs

```diff
@@ -912,7 +912,7 @@ impl Operand {
     }
 
     /// If this operand is for a fixed non-allocatable register (see
-    /// [`Operand::fixed`]), then returns the physical register that it will
+    /// [`OperandConstraint::FixedReg`]), then returns the physical register that it will
```

Comment:
This is incorrect (`FixedReg` is not the only aspect of "fixed non-allocatable" operands -- they have a sentinel vreg value too) -- the function reference was stale though. Could you change to `Operand::fixed_nonallocatable`?

Reply:
Fixed in 06850ff5b7d3ee40da961155749ee049eb74445f :+1: